designer/server/src/main/resources/defaultDesignerConfig.conf

```diff
@@ -227,6 +227,8 @@ usageStatisticsReports {
   source: ${?USAGE_REPORTS_SOURCE}
 }
 
+statisticsPublicKey: "MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAjdvfEFH8jBF56UZmtQvUH1gUCnRkHTke/jnOEIjdSmSqoyWGJ9UKC/PgLYyqLqNRKq2eEmLr26tQRaIoOHLcdGlGZXIrRkWHAPH8QTGrDt4Qm/COB8BPS7oV2tATsUN7zJNWfVRNDzcunzDwAtZKs4SDTsFLAPrZ5CKMt5JK9Q7Xrzekl5PunzJEuIOmlWusFanIqCgs0d245NVRKhbkqq/JkoseEB4sDXFwNyO7sO51aLgDST+P/G+tPveQpMusbGBK48Syce93R6bTf0Bd8KGaJQWvJPjD6rbX+K3vSQHqgb+NjIyUAWWNHuFvRt2NQ5iHsuaBU0B/61W8y6oMxQIDAQAB"
```

Comment:
We cannot put it to the configuration, because user can change configuration. What we need is to create static files in resources and then read it with `Class.getResource("/static_file_name.key")`